src/controllers/movie.js

```diff
@@ -0,0 +1,119 @@
+const { Prisma } = require("@prisma/client")
+const prisma = require('../utils/prisma')
+
+const getAllMovies = async (req, res) => {
+    try {
+        if (Object.keys(req.query).length === 0) {
+            const allMovies = await prisma.movie.findMany({ include: { screenings: true } })
+            res.status(200).json({ movies: allMovies })
+        } else {
+            const allMovies = await prisma.movie.findMany({
+                where: {
+                    AND: [Object.keys(req.query)[0] === "runtimeLt" ?
+                        { runtimeMins: { lte: +req.query.runtimeLt } } :
+                        { runtimeMins: { gte: +req.query.runtimeGt } }
+                    ]
+                },
+                include: {
+                    screenings: true,
+                }
+            })
+            res.status(200).json({ movies: allMovies })
+        }
```

Comment:
A couple of notes on this:

1. You can eliminate the `else` completely if you do `return res.status(200).json({ movies: allMovies })` on line 8, which will clean things up a little

2. Although fun to write, the general rule for ternary is "if it doesn't fit on one line, it should be an if statement" - this is very likely to get a PR rejected in a work environment